context.go

```diff
@@ -339,3 +463,21 @@ func (c *context) resolveXAttrs(fp string, fi os.FileInfo, base *resource) (map[
 
 	return nil, nil
 }
+
+// AbsoluteSymlinkPath turns the symlink target into absolute paths from
+// the given root.
+func AbsoluteSymlinkPath(root, linkname, target string) (string, error) {
+	return filepath.Join(root, target), nil
+}
+
+// RelativeSymlinkPath turns the symlink target into a relative path
+// from the given linkname.
+func RelativeSymlinkPath(root, linkname, target string) (string, error) {
+	return filepath.Rel(filepath.Join(root, linkname), target)
+}
+
+// ChrootSymlinkPath uses the given absolute target intended for a
+// chroot environment using the given root.
+func ChrootSymlinkPath(root, linkname, target string) (string, error) {
+	return target, nil
+}
```

Comment:
Currently only `AbsoluteSymlinkPath` is used, the others are added to demonstrate how it may be used in the future. It would involve having the context configurable. For a namespace version, not sure if additional properties would be needed, as we previously discussed the namespace case would ideally use the `AbsoluteSymlinkPath` and the re-map would not be the responsibility of continuity.